src/core/shape_inference/include/fft_base_shape_inference.hpp

```diff
@@ -53,14 +53,23 @@ void shape_infer(const ov::op::util::FFTBase* op,
         // [n_0, ..., n_{r - 1}, 2] is interpreted as a complex tensor with the shape
         // [n_0, ..., n_{r - 1}].
         if (axes_shape.rank().is_static() && axes_are_known) {
+            ov::AxisSet axes_set;
             for (int64_t& axis : axes) {
+                NODE_VALIDATION_CHECK(op,
```

Comment:
Due to complex input represented like {D0, D1, D2, .., 2} (the last dim represents real and imaginary part), axes should be in range  `axis < input_rank-1`

Reply:
Yeah, we have check for this few lines below - https://github.com/openvinotoolkit/openvino/blob/5177db82e0c4af70ca8b569388eb04455198ebf3/src/core/shape_inference/include/fft_base_shape_inference.hpp#L79
But we can merge them together.

Reply:
done